### PR TITLE
2.0.0 Server Role Update

### PR DESCRIPTION
**Pull request by pyrodie18**
##### SUMMARY
Initial changes to server role for 2.0.0

##### COMPONENT NAME
Server Role

##### ADDITIONAL INFORMATION
Major Changes include:

- Updating Molecule matrix to test all OS, with all versions, and all databases (#777 )
- Updated documentation
- Removed support for anything < 5.0 (#766)
- Cleaned up handling of variables between OSs
- Standardized tasks names
- Added Ubuntu 22.04 (#763)
- Added Support for Debian 11 (#764)
- Removed support for EOL operating systems
- Cleaned up some variables (#768 )
- Removed unnecessary installs (#696)

### CODE REVIEW

File: docs/ZABBIX_SERVER_ROLE.md [hunk 1]
```diff
@@ -57,15 +57,15 @@ ansible-galaxy collection install ansible.posix
 
 ### MySQL
 
-When you are a MySQL user and using Ansible 2.10 or newer, then there is a dependency on the collection named `community.mysql`. This collections are needed as the `mysql_` modules are now part of collections and not standard in Ansible anymmore. Installing the collection:
+When you are a MySQL there is a dependency on the collection named `community.mysql`.  Installing the collection:
```
Comment (mu1f407) [inline docs/ZABBIX_SERVER_ROLE.md:60]:
```suggestion
When you are using a MySQL there is a dependency on the collection named `community.mysql`.  Installing the collection:
```

File: docs/ZABBIX_SERVER_ROLE.md [hunk 2]
```diff
@@ -57,15 +57,15 @@ ansible-galaxy collection install ansible.posix
...
 
 ```sh
 ansible-galaxy collection install community.mysql
 ```
 
 ### PostgreSQL
 
-When you are a PostgreSQL user and using Ansible 2.10 or newer, then there is a dependency on the collection named `community.postgresql`. This collections are needed as the `postgresql_` modules are now part of collections and not standard in Ansible anymmore. Installing the collection:
+When you are a PostgreSQL there is a dependency on the collection named `community.postgresql`. Installing the collection:
```
Comment (mu1f407) [inline docs/ZABBIX_SERVER_ROLE.md:68]:
```suggestion
When you are using a PostgreSQL there is a dependency on the collection named `community.postgresql`. Installing the collection:
```

File: docs/ZABBIX_SERVER_ROLE.md [hunk 3]
```diff
@@ -75,23 +75,16 @@ ansible-galaxy collection install community.postgresql
...
-| macOS 10.14         |     |     |     |     |           | V   | V         |           |
+| Zabbix              | 6.2 | 6.0 (LTS) | 5.0 (LTS) | 
+|---------------------|-----|-----------|-----------|
+| Red Hat Fam 8       |  V  |  V        |  V        |
+| Red Hat Fam 7       |     |           |  V        |
+| Ubuntu 22.04 jammy  |  V  |  V        |  V        |
+| Ubuntu 20.04 focal  |  V  |  V        |  V        |
+| Ubuntu 18.04 bionic |     |  V        |  V        |
+| Debian 11 bullseye  |  V  |  V        |  V        |
+| Debian 10 buster    |     |  V        |  V        ?
```
Comment (mu1f407) [inline docs/ZABBIX_SERVER_ROLE.md:86]:
```suggestion
| Debian 10 buster    |     |  V        |  V        |
```

**Review by mu1f407 — COMMENTED** [review]
Found just some small things in the docs. Overall very nice update and a lot of cleanups. Thank you for your work @pyrodie18.

File: docs/ZABBIX_SERVER_ROLE.md (hunk 3 above)
Comment (pyrodie18) [reply docs/ZABBIX_SERVER_ROLE.md]: Changes made

File: docs/ZABBIX_SERVER_ROLE.md (hunk 2 above)
Comment (pyrodie18) [reply docs/ZABBIX_SERVER_ROLE.md]: Changes made

**Review by D3DeFi — APPROVED** [review]
I think this looks good. Went through pretty quickly though so may not spotted some typos or something

**Review by D3DeFi — APPROVED** [review]

**Pull request merged**